services/npm/npm-type-definitions.service.js

```diff
@@ -7,7 +7,7 @@ const NpmBase = require('./npm-base')
 // package.json.
 module.exports = class NpmTypeDefinitions extends NpmBase {
   static get category() {
-    return 'other'
+    return 'platform-support'
```

Comment:
I don't think "Platform & Version Support" is the ideal name for this grouping, though I do feel the Flow/TS badge belongs in it.